Feature/fix typos (#735)
smart-data-lake/smart-data-lake: docs/getting-started/part-1/get-departures.md

After executing it, you will see the file *data/stg_departures/result.json* has been replaced with the output of your pipeline.

:::caution
Since both web servers are freely available on the internet, **a rate limiting applies**. https://opensky-network.org/ will stop responding if you make too many calls.

> If the download fails because of a timeout, wait a couple of minutes and try again. In the worst case, it should work again on the next day.

Reviewer: Sounds a bit strange IMHO, "in the worst case, it should work again".
```suggestion
If the download fails because of a timeout, wait a couple of minutes and try again. In the worst case, it will only work again the following day.
```
